tests/unit_tests/grid/test_face_functions.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import pytest
+
+from resqpy.grid import Grid
+from resqpy.model import Model
+
+import resqpy.grid.face_functions as ff
+
+
+def test_face_centre(basic_regular_grid: Grid):
+    # Arrange
+    cell = None
+    axis = 0
+    zero_or_one = 0
+    expected_face_centres = np.array([[[[50.0, 25.0, 0.0], [150.0, 25.0, 0.0]], [[50.0, 75.0, 0.0], [150.0, 75.0,
+                                                                                                     0.0]]],
+                                      [[[50.0, 25.0, 20.0], [150.0, 25.0, 20.0]],
+                                       [[50.0, 75.0, 20.0], [150.0, 75.0, 20.0]]]])
+
+    # Act & Assert
+    face_centre = ff.face_centre(basic_regular_grid, cell, axis, zero_or_one)
+    np.testing.assert_array_equal(face_centre, expected_face_centres)
+
+
+def test_face_centre_cell_kji0(basic_regular_grid: Grid):
+    # Arrange
+    cell = (0, 0, 0)
+    axis = 0
```

Comment:
It might be good to also test for other values of axis and zero_or_one (face polarity)